operator/config/olm-catalog/ibm-spectrum-scale-csi-operator/2.5.0/ibm-spectrum-scale-csi-operator.v2.5.0.clusterserviceversion.yaml

```diff
@@ -1,295 +1,342 @@
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "CSIScaleOperator",
+          "metadata": {
+            "labels": {
+              "app.kubernetes.io/instance": "ibm-spectrum-scale-csi-operator",
+              "app.kubernetes.io/managed-by": "ibm-spectrum-scale-csi-operator",
+              "app.kubernetes.io/name": "ibm-spectrum-scale-csi-operator",
+              "release": "ibm-spectrum-scale-csi-operator"
+            },
+            "name": "ibm-spectrum-scale-csi",
+            "namespace": "ibm-spectrum-scale-csi-driver"
+          },
+          "spec": {
+            "attacherNodeSelector": [
+              {
+                "key": "scale",
+                "value": "true"
+              }
+            ],
+            "clusters": [
+              {
+                "id": "\u003c Primary Cluster ID - WARNING - THIS IS A STRING NEEDS YAML QUOTES! \u003e",
+                "primary": {
+                  "primaryFs": "\u003c Primary Filesystem \u003e"
+                },
+                "restApi": [
+                  {
+                    "guiHost": "\u003c Primary cluster GUI IP/Hostname \u003e"
+                  }
+                ],
+                "secrets": "secret1",
+                "secureSslMode": false
+              }
+            ],
+            "pluginNodeSelector": [
+              {
+                "key": "scale",
+                "value": "true"
+              }
+            ],
+            "provisionerNodeSelector": [
+              {
+                "key": "scale",
+                "value": "true"
+              }
+            ],
+            "resizerNodeSelector": [
+              {
+                "key": "scale",
+                "value": "true"
+              }
+            ],
+            "snapshotterNodeSelector": [
+              {
+                "key": "scale",
+                "value": "true"
+              }
+            ]
+          },
+          "status": {}
+        }
+      ]
     capabilities: Basic Install
     categories: Storage
-    certified: 'false'
+    certified: "false"
     containerImage: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v2.5.0
-    createdAt: Mon Dec 20 14:55:05 IST 2021
+    createdAt: Mon Nov 01 13:55:05 IST 2021
```

Comment:
Date needs to be Mon Dec 20 14:55:05 IST 2021

Reply:
fixed